e2e/app.e2e-spec.ts

```diff
@@ -5,10 +5,10 @@ describe('gamma-sky App', function() {
 
   beforeEach(() => {
     page = new GammaSkyPage();
-  })
+  });
 
   it('should display message saying app works', () => {
     page.navigateTo();
-    expect(page.getParagraphText()).toEqual('gamma-sky works!');
+    expect(page.getParagraphText()).toEqual('app works!');
```

Comment:
Try running `ng e2e` locally. Does it pass? What are these? Should we run them on travis-ci?